aws/aws-parallelcluster-cookbook
# [RHEL8] Transform fabric manager into a resource

### Description of changes
* Transform fabric manager into a resource
* Remove some value_for_platform
* Add testing for fabric_manager
* Add testing for cuda redhat8
* Move defaults under aws-parallelcluster-common

### Tests
* Manually tested on ec2 using 
```
KITCHEN_GLOBAL_YAML=kitchen.global.yml
KITCHEN_YAML=kitchen.ec2.yml
KITCHEN_LOCAL_YAML=kitchen.recipes-install.yml


kitchen test nvidia -c5 --parallel
```

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
